osu.Game/Beatmaps/BeatmapInfo.cs

```diff
@@ -2,180 +2,195 @@
+        [MapTo(nameof(Status))]
+        public int StatusInt { get; set; } = (int)BeatmapOnlineStatus.None;
 
-        [NotMapped]
-        public int? MaxCombo { get; set; }
+        [Indexed]
+        public int OnlineID { get; set; } = -1;
 
-        /// <summary>
-        /// The playable length in milliseconds of this beatmap.
-        /// </summary>
         public double Length { get; set; }
 
-        /// <summary>
-        /// The most common BPM of this beatmap.
-        /// </summary>
         public double BPM { get; set; }
 
-        public string Path { get; set; }
+        public string Hash { get; set; } = string.Empty;
+
+        public double StarRating { get; set; }
 
-        [JsonProperty("file_sha2")]
-        public string Hash { get; set; }
+        public string MD5Hash { get; set; } = string.Empty;
 
         [JsonIgnore]
         public bool Hidden { get; set; }
 
-        /// <summary>
-        /// MD5 is kept for legacy support (matching against replays, osu-web-10 etc.).
-        /// </summary>
-        [JsonProperty("file_md5")]
-        public string MD5Hash { get; set; }
+        #region Properties we may not want persisted (but also maybe no harm?)
 
-        // General
         public double AudioLeadIn { get; set; }
-        public float StackLeniency { get; set; } = 0.7f;
-        public bool SpecialStyle { get; set; }
 
-        public int RulesetID { get; set; }
+        public float StackLeniency { get; set; } = 0.7f;
 
-        public RulesetInfo Ruleset { get; set; }
+        public bool SpecialStyle { get; set; }
 
         public bool LetterboxInBreaks { get; set; }
+
         public bool WidescreenStoryboard { get; set; }
+
         public bool EpilepsyWarning { get; set; }
 
-        /// <summary>
-        /// Whether or not sound samples should change rate when playing with speed-changing mods.
-        /// TODO: only read/write supported for now, requires implementation in gameplay.
-        /// </summary>
         public bool SamplesMatchPlaybackRate { get; set; }
 
-        public CountdownType Countdown { get; set; } = CountdownType.Normal;
-
-        /// <summary>
-        /// The number of beats to move the countdown backwards (compared to its default location).
-        /// </summary>
-        public int CountdownOffset { get; set; }
-
-        [NotMapped]
-        public int[] Bookmarks { get; set; } = Array.Empty<int>();
-
         public double DistanceSpacing { get; set; }
+
         public int BeatDivisor { get; set; }
+
         public int GridSize { get; set; }
-        public double TimelineZoom { get; set; }
 
-        // Metadata
-        [Column("Version")]
-        public string DifficultyName { get; set; }
+        public double TimelineZoom { get; set; }
 
-        [JsonProperty("difficulty_rating")]
-        [Column("StarDifficulty")]
-        public double StarRating { get; set; }
+        [Ignored]
+        public CountdownType Countdown { get; set; } = CountdownType.Normal;
```

Comment:
Was it intentional leaving this as-is? sounds like it could be worked around in a similar fashion to `Status`/`StatusInt`.

Reply:
Yeah, explained in https://github.com/ppy/osu/pull/16428/commits/d7fe3584cde52011d776d26f1bcb95a651d56896.